app/src/controllers/administrate/projects/actionCreators.js

```diff
@@ -0,0 +1,16 @@
+import { getStorageItem, setStorageItem } from 'common/utils';
+import { FETCH_PROJECTS, TABLE_VIEW, USER_VIEW, TOGGLE_PROJECTS_VIEW } from './constants';
+
+export const fetchProjectsAction = (params) => ({
+  type: FETCH_PROJECTS,
+  payload: params,
+});
+
+export const toggleViewAction = (type) => (dispatch) => {
+  const userView = getStorageItem(USER_VIEW) || TABLE_VIEW;
```

Comment:
Isn't grid should be default view?

Reply:
on Dashboard Table View is the default, probably should be the same here?

Reply:
I think it's better to clarify this with BA